ServiceWebsite/ServiceWebsite.AcceptanceTests/Helpers/TestLogger.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Reflection;
+using ServiceWebsite.Common;
+
+namespace ServiceWebsite.AcceptanceTests.Helpers
+{
+    public class TestLogger
+    {
+        public static void Log(string  eventTitle, string message)
+        {
+            var category = Assembly.GetCallingAssembly().GetName().Name;
+            Console.WriteLine("Logging {0}: {1}: {2} ", category, eventTitle, message);
```

Comment:
Don't think you need the word "Logging"....
Also use string interpolation e.g. $"{category}: {eventTitle}: {message} "

Reply:
done